src/binaryen-c.h

```diff
@@ -2952,6 +2952,13 @@ BINARYEN_API bool BinaryenGetMemorySegmentPassive(BinaryenModuleRef module,
 BINARYEN_API void BinaryenCopyMemorySegmentData(BinaryenModuleRef module,
                                                 const char* segmentName,
                                                 char* buffer);
+BINARYEN_API void BinaryenAddDataSegment(BinaryenModuleRef module,
+                                         const char* memoryName,
+                                         const char* segmentName,
+                                         const char* segmentData,
+                                         bool segmentPassive,
+                                         BinaryenExpressionRef segmentOffset,
+                                         BinaryenIndex segmentSize);
```

Comment:
Perhaps it would be clearer to reorder `segmentData` next to `segmentSize`? In general matching the order of `wasm-builder`'s `makeDataSegment` seems the most predictable.

Reply:
> Perhaps it would be clearer to reorder `segmentData` next to `segmentSize`? In general matching the order of `wasm-builder`'s `makeDataSegment` seems the most predictable.

@kripken, I have made corresponding changes and you can check that out :)